lib/core_modules/module/dependablelist.js

```diff
@@ -1,58 +1,60 @@
+'use strict';
```

Comment:
Nothing to change now, but is this no longer necessary if we require Node 6? Is there an es6 improvement that makes this unnecessary going forward?